mcv/hostname.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def _read_stripped_lines(f):
+    return [l.rstrip() for l in f.readlines()]
+
+
+def _splice_alias(alias, line):
```

Comment:
I don't understand what's going on here--"aliasing" is a generic operation that should be applicable to any IP, not just localhost